src/java.base/share/classes/java/lang/ScopedValue.java

```diff
@@ -394,19 +394,20 @@ public <T> T get(ScopedValue<T> key) {
          * to its value in the current thread.
          * When the operation completes (normally or with an exception), each scoped value
          * in the mapping will revert to being unbound, or revert to its previous value
-         * when previously bound, in the current thread.
+         * when previously bound, in the current thread. If {@code op} completes with an
+         * exception then it propagated by this method.
          *
-         * <p> Scoped values are intended to be used in a <em>structured manner</em>.
-         * If {@code op} creates a {@link StructuredTaskScope} but does not {@linkplain
-         * StructuredTaskScope#close() close} it, then exiting {@code op} causes the
-         * underlying construct of each {@code StructuredTaskScope} created in the
-         * dynamic scope to be closed. This may require blocking until all child threads
-         * have completed their sub-tasks. The closing is done in the reverse order that
-         * they were created. Once closed, {@link StructureViolationException} is thrown.
+         * <p> Scoped values are intended to be used in a <em>structured manner</em>. If code
+         * invoked directly or indirectly by the operation creates a {@link StructuredTaskScope}
+         * but does not {@linkplain StructuredTaskScope#close() close} it, then it is detected
+         * as a <em>structure violation</em> when the operation completes (normally or with an
+         * exception). In that case, the underlying construct of the {@code StructuredTaskScope}
+         * is closed and {@link StructureViolationException} is thrown.
```

Comment:
Hello Alan, does this mean that the original exception (if any) that caused the completion of the operation would be lost and the caller would instead receive just the `StructureViolationException`?

Reply:
No, it's just means the SVE is suppressed - see the discussion in the CSR.

Reply:
Thank you, looks good to me.